src/controllers/user.py

```diff
@@ -22,14 +22,17 @@ def create_user(body):
     return result, code
 
 
-def get_all_users():
+def get_all_users(filter=None):
+    if filter:
+        result, code = db.get_all(User, {"invalid": 5})
+        return result, code
+
     result, code = db.get_all(User)
-    if result:
-        if code == 200:  # if successful, returns the data
-            users = [u.to_dict() for u in result]  # converts rows to dict
-            return users, code
-        return result, code  # else, returns database error and error code
-    return [], 200
+
```

Comment:
it is necessary to return to an empty list with code 200 when there is no record in the bank and when there was no error